internal/provider/resource_access_model.go

```diff
@@ -235,19 +236,27 @@ func (a *accessModelResource) Update(ctx context.Context, req resource.UpdateReq
 	anyChange := false
 
 	// items that could be changed
-	var newAccess string
+	access := state.ID.ValueString()
```

Comment:
why do we start with access as "user:admin"? which at line 268 would change to "admin" or something else?

If I add a user to the access after start without changing access you get:
```
juju_access_model.one: Modifying... [id=machinetest:admin:heather]
juju_access_model.one: Modifications complete after 0s [id=machinetest:machinetest:admin:heather:heather,marie]
```